src/demo/pages/TooltipPage/examples/Tooltip.Basic.Example.tsx

```diff
@@ -0,0 +1,53 @@
+/* tslint:disable:no-unused-variable */
+import * as React from 'react';
+/* tslint:enable:no-unused-variable */
+
+import {
+  Tooltip,
+  Button
+} from '../../../../index';
+
+export interface ITooltipBasicExampleState {
+  isTooltipVisible?: boolean;
+}
+
+export class TooltipBasicExample extends React.Component<any, ITooltipBasicExampleState> {
+  private _menuButtonElement: HTMLElement;
+
+  public constructor() {
+    super();
+
+    this._onDismiss = this._onDismiss.bind(this);
+
+    this.state = {
+      isTooltipVisible: false,
+    };
+  }
+
+  public render() {
+    let { isTooltipVisible } = this.state;
+
+    return (
+      <div className='ms-TooltipExample'>
```

Comment:
@joschect Maybe you could think about this as well. If we made Callout take "content" as a prop, then we could wrap children and not require "targetElement"